netci.groovy

```diff
@@ -1741,6 +1742,11 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                         case 'illink':
                             Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} ${configuration} via ILLink", "(?i).*test\\W+${os}\\W+${architecture}\\W+${configuration}\\W+${scenario}.*")
                             break
+                        case 'corefx_innerloop':
+                            if (configuration == 'Release') {
```

Comment:
Do we want this for release? Generally we prefer checked.

Reply:
I think both Checked and Release are appropriate.

---
In reply to: [196822183](https://github.com/dotnet/coreclr/pull/18388#discussion_r196822183) [](ancestors = 196822183)